### DISCUSSION TITLE
Private endpoint

### DISCUSSION THREAD

**Discussion by mjdovalina** [mspnp/AzureNamingTool#107] [Q&A] [answered]
Is anyone working on adding private endpoint to the naming tool?

**Comment by jamasten** [comment, marked as answer]
Hi @mjdovalina. Yes, my goal is to add anything missing from the [Naming rules and restrictions for Azure resources](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules) webpage to the Azure Naming Tool next month. Private endpoints are listed on that webpage now, so I'll be able to add that resource type. We rely on the CAF team to provide the restrictions in the documentation then we develop the regular expressions and rules in the tool for the resource types. This tool is managed by engineers in their spare time and is a best effort. If you have the skills, please feel to submit PRs to the repo with the appropriate data updates. Thank you for using the tool!

**Reply by mjdovalina** [reply]
Thanks so much for such a quick response! I'm grateful for all of the work you all do. It's been more than helpful!
